docs/operations/annotations_en.md

```diff
@@ -0,0 +1,96 @@
+# Annotations for the k8s-dogu-operator
+
+This document contains the detailed description of all generated annotation of the k8s-dogu-operator.
+
+## Services
+
+This section contains all annotation that are appended to K8s services, if necessary.
```

Comment:
```suggestion
This section contains all annotations that are appended to K8s services, if necessary.
```

Reply:
The English document is automatically generated.

Reply:
And this is the reason you should double-check it :D